### PR TITLE
dist: version necessary marks

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
- Bump %graph-update to v3
- Version glob, treaty, docket, kiln-vats-diff marks

### CODE REVIEW

File: pkg/landscape/app/graph-store.hoon [hunk 1]
```diff
@@ -111,7 +112,7 @@
   ?>  (team:title our.bowl src.bowl)
   =^  cards  state
     ?+  mark           (on-poke:def mark vase)
-      %graph-update-2  (graph-update !<(update:store vase))
+      %graph-update-3  (graph-update !<(update:store vase))
```
Comment (Fang-) [inline pkg/landscape/app/graph-store.hoon:115]: Are these not backwards compatible in some way? What changed?

Comment (belisarius222) [reply pkg/landscape/app/graph-store.hoon:115]: references to other apps were added

Comment (Fang-) [reply pkg/landscape/app/graph-store.hoon:115]: We can and should continue accepting `%graph-update-2` then, right?

**Comment by Fang-** [comment]
Also what about `/mar/treaties`?

**Pull request merged**